caf-audit-binding-webservice/src/main/java/com/hpe/caf/auditing/webserviceclient/WebServiceClientAuditEventBuilder.java

```diff
@@ -284,7 +287,15 @@ private String getAuditEventAsJsonString() throws IOException {
                             .value(auditEventParam.getParamIndexingHint().toString().toLowerCase());
                 }
                 jsonWriter.name("paramValue");
-                gson.toJson(auditEventParam.getParamValue(), Object.class, jsonWriter);
+                if(auditEventParam.getParamValue() instanceof Date){
+                    final Date date = (Date)auditEventParam.getParamValue();
+                    DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+                    df.setTimeZone(TimeZone.getTimeZone("UTC"));
+                    jsonWriter.value(df.format(date));
+                }
+                else {
+                    gson.toJson(auditEventParam.getParamValue(), Object.class, jsonWriter);
+                }
```

Comment:
It's pretty worrying that the web service code has had to change to do the Elasticsearch upgrade.

Reply:
Yes, the release build failure for SCMOD-6230 was timezone related. This change to the web service client is no longer needed.